README.md

```diff
@@ -30,6 +32,34 @@ queue.on('processed', function(err, res, item) {
 queue.start();
 ```
 
+### Options
+
+The queue can be initialized with the following options (*defaults shown*):
+
+```js
+var options = {
+  ackTimer: 1000,        // ack interval in ms
```

Comment:
The ack/reclaim settings are pretty advanced settings that are nearly impossible to get right. I'm honestly not sure why I made them configurable in the first place, but we probably shouldn't be suggesting people can modify them.